packages/react-tooltip/src/components/Tooltip/Tooltip.types.ts

```diff
@@ -117,14 +101,18 @@ export type OnVisibleChangeData = {
  * Properties for Tooltip
  */
 export type TooltipProps = ComponentProps<TooltipSlots> &
-  Partial<Omit<TooltipCommons, 'content' | 'relationship'>> &
-  Pick<TooltipCommons, 'content' | 'relationship'>;
+  Partial<Omit<TooltipCommons, 'relationship'>> &
+  Pick<TooltipCommons, 'relationship'> & {
+    children?: (React.ReactElement & { ref?: React.Ref<unknown> }) | ((props: TooltipTriggerProps) => React.ReactNode);
+  };
```

Comment:
Just to double check, is `content` still a required prop after this change?

Reply:
Fixed in 394f904, as discussed in offline: it's not ideal, but works for now.